src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
Thanks for the tests Jack. I'll take a look later. 

Do not disable gcount: that needs to be unique for each gate for security.